autoload-bootstrap.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * Autoload the autoload files of the individual standards.
+ *
+ * @category PHP
+ * @package  YoastCS
+ * @author   Jip Moors
+ */
+
+if ( file_exists( __DIR__ . '/vendor/wimg/php-compatibility/PHPCSAliases.php' ) ) {
```

Comment:
Side-note/Teaching moment: the PHPCompatibility alias file would not be needed anyway as - again - their own ruleset loads it.

Basically, the only time when you need to load the alias files of another standard is when you just load sniffs from another standard, not the standard itself.

So, for the QA-WP project, I load them as those rulesets don't include the whole WP standard, but only individual sniffs.
For the unit tests you need to load them as the sniffs are tested individually, independent of the ruleset.